ext/nmatrix/storage/list/list.cpp

```diff
@@ -602,6 +602,14 @@ void init_default(LIST_STORAGE* s) {
   s->default_val = NM_ALLOC(D);
   *reinterpret_cast<D*>(s->default_val) = 0;
 }
+  
+template <typename DType>
+bool is_symmetric(const LIST_STORAGE* mat) {
+  
+  std::cout << "\n\n\nImplementation goes here\n\n\n";
+
+  return true;
+}
```

Comment:
How should I implement this? Should I enumerate over the elements of linked list or what?